File: WarehouseManager.HandheldApp/handheldapp/lib/models/CreatePalletDetailVm.dart
class CreatePalletDetailVm {
  String PO = '';
  String? Item;
  String PalletNo = '';
  int Quantity = 0;

  CreatePalletDetailVm({
    required this.PO,
    required this.Item,
    required this.PalletNo,
    required this.Quantity,
  });

  Map<String, dynamic> toJson() {
    return {'PO': PO, 'Item': Item, 'PalletNo': PalletNo, 'Quantity': Quantity};
  }
}

File: WarehouseManager.ViewModels/Warehouse/PalletDetail/CreatePalletDetailVm.cs
namespace WarehouseManager.ViewModels.Warehouse.PalletDetail
{
    public class CreatePalletDetailVm
    {
        public string PO { get; set; } = string.Empty;
        public string? Item { get; set; }
        public string PalletNo { get; set; } = string.Empty;
        public int Quantity { get; set; }
    }
}